ntbs-service/DataMigration/TreatmentEventMapper.cs

```diff
@@ -78,5 +80,31 @@ private async Task TryAddTbServiceAndCaseManagerToTreatmentEvent(TreatmentEvent
                 ev.CaseManagerId = (await _referenceDataRepository.GetUserByUsernameAsync(caseManagerUsername)).Id;
             }
         }
+
+        private string RemoveUnnecessaryNoteInfo(string note)
+        {
+            if (note.IsNullOrEmpty())
+            {
+                return null;
+            }
+            var trimmedNote = note.Trim();
+            var mainPattern =
+                @"(Dear|Hi)[0-9a-zA-Z -]+,{0,1}[\n\r ]*(?<caseManagerText>[0-9a-zA-Z \/\-—,.'`@#&+;:$_()<>\\\[\]=\*\?\n\r]*)"
+                + @"(?<uselessInfo>Id: [0-9 ]+[\n\r ]*Patient: [a-zA-Z -]+[\n\r ]*Case report date: [0-9 ]{2}\/[0-9]{2}\/[0-9]{4})"
+                + @"(?<appendedNote>[0-9a-zA-Z \/\-—,.'`@#&+;:$_()<>\\\[\]=\*\?\n\r]*)";
+            var caseManagerPattern = @"You have been identified as the new case manager for the case below\.[\n\r]*";
```

Comment:
Is the number of new lines at the end of this sentence really variable? Could there also be spaces?

Reply:
From what I've seen it is usually between 0 and 3, but it's a mix of \r\n and just \n.
I don't think it's worth not matching just because there's 4 newlines.